Assets/Scripts/Puzzle/ItemPlace.cs

```diff
@@ -32,7 +32,11 @@ private void Update()
 
         void ShowMesh()
         {
-            gameObject.GetComponent<MeshRenderer>().enabled = true;
+            var mesh = gameObject.GetComponent<MeshRenderer>();
+            if (mesh != null)
+            {
+                mesh.enabled = true;
+            }
```

Comment:
its better to have MeshItemPlace and ItemPlace so this null check would not be needed in that, but its okay for now